### PR TITLE
Set up unit tests

### PR DESCRIPTION
**Pull request by ghost**
[PT Story](https://www.pivotaltracker.com/story/show/171811162)
```
As a development team,
In order to unit test our code
We would like to set up Rspec
```
## Changes proposed in this pull request:

* Adds rspec for testing.
* Adds factory bot and shoulda matchers.

## What I have learned working on this feature:

* Refreshed knowledge about installing testing framework

## Packages/Gems added

- gem 'rspec-rails'
- gem 'shoulda-matchers'
- gem 'factory_bot_rails'

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -60,9 +60,16 @@ GEM
       msgpack (~> 1.0)
     builder (3.2.4)
     byebug (11.1.1)
```
Comment (robin-lillqvist) [inline Gemfile.lock]: we're not using byebug, so we should remove the gemfile.lock file and re-bundle.

**Review by robin-lillqvist — CHANGES_REQUESTED** [review]
one comment

File: Gemfile.lock (hunk 1 above)
Comment (ghost) [reply Gemfile.lock]: Done

**Review by robin-lillqvist — APPROVED** [review]
Approved

**Review by robin-lillqvist — APPROVED** [review]
LGTM

**Comment by robin-lillqvist** [comment]
Merged

**Pull request merged**